docs/index.rst

```diff
@@ -0,0 +1,20 @@
+.. Fromager documentation master file, created by
+   sphinx-quickstart on Fri Jul 26 11:06:15 2024.
+   You can adapt this file completely to your liking, but it should at least
+   contain the root `toctree` directive.
+
+Fromager documentation
+======================
+
+Add your content using ``reStructuredText`` syntax. See the
+`reStructuredText <https://www.sphinx-doc.org/en/master/usage/restructuredtext/index.html>`_
+documentation for details.
+
+
+.. toctree::
+   :maxdepth: 2
+   :caption: Contents:
+
+.. click:: src.fomager.commands.bootstrap:bootstrap
```

Comment:
The instruction expects a Python import name to the root click group. For Fromager it's `from fromager.__main__ import main`:

```suggestion
.. click:: fromager.__main__:main
```

Reply:
I changed this and now I get the following error:

```
/home/rdevasth/fromager/docs/cli.rst:1: ERROR: Failed to import "main" from "fromager.__main__". The following exception was raised:
Traceback (most recent call last):
  File "/home/rdevasth/fromager/venv/lib/python3.11/site-packages/sphinx_click/ext.py", line 444, in _load_module
    mod = __import__(module_name, globals(), locals(), [attr_name])
          ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
ModuleNotFoundError: No module named 'fromager'
```

Reply:
Do you have fromager installed? Sphinx is going to try to import the source code to generate those docs.

One convenient way to do that is to use tox instead of make. Tox lets you define a virtualenv with dependencies and then a set of commands to run. [This](https://github.com/sphinx-contrib/spelling/blob/main/tox.ini#L72) is what I use for generating documentation for another library that I maintain, for example.